torch/nn/init.py

```diff
@@ -5,11 +5,39 @@
 from torch.autograd import Variable
 
 
+def calculate_gain(nonlinearity, param=None):
+    """Return the recommended gain value for the given nonlinearity function.
+
+    Args:
+        nonlinearity: the nonlinear function (`nn.functional` name)
+        param: optional parameter for the nonlinear function
+
+    Examples:
+        >>> gain = nn.init.gain('lrelu')
+    """
+    if gain == 'sigmoid':
+        return 1
+    elif gain == 'tanh':
+        return 5 / 3
```

Comment:
I think it might be better to use `5. / 3`, as Python 2.7 might not handle the casting to float